CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ IMPROVEMENTS:
 
         let g:go_list_type_commands = {"GoBuild": "quickfix", "GoTest": "locationlist"}
 
+* Disable `g:go_autodetect_gopath` by default [[GH-1461]](https://github.com/fatih/vim-go/pull/1461).
```

Comment:
Can you move this up to the latest changelog? This seems to be in the old place as we released a new version.